server/Arcadia.Assistant.Calendar/SickLeave/SendEmailSickLeaveActor.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Arcadia.Assistant.Calendar.SickLeave
+{
+    using System;
+    using Akka.Actor;
+    using Akka.DI.Core;
+    using MimeKit;
+    using Configuration.Configuration;
+    using MailKit.Net.Smtp;
+    using MailKit.Security;
+    using Events;
+
+    public class SendEmailSickLeaveActor : UntypedActor
+    {
+        private readonly IEmailSettings mailConfig;
+        private readonly ISmtpSettings smtpConfig;
+        public SendEmailSickLeaveActor(IEmailSettings mailConfig, ISmtpSettings smtpConfig)
+        {
+            this.mailConfig = mailConfig;
+            this.smtpConfig = smtpConfig;
+        }
+
+        public static Props GetProps() => Context.DI().Props<SendEmailSickLeaveActor>();
+
+        protected override void OnReceive(object message)
+        {
+            switch (message)
+            {
+                case SickLeaveIsApproved ev:
+                    SendEmail(ev);
```

Comment:
have you installed Resharper? :) it should complain on lack of `this.` - just a code style issue.